internal/autoupdate/autoupdate_sync.go

```diff
@@ -0,0 +1,222 @@
+package autoupdate
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/google/go-github/v35/github"
+	"go.uber.org/zap"
+
+	"github.com/simplesurance/goordinator/internal/logfields"
+)
+
+type syncAction int
+
+const (
+	undefined syncAction = iota
+	enqueue
+	dequeue
+)
+
+// Sync synchronized the states of the autoupdater queues with the current
+// pull-request state at GitHub.
```

Comment:
```suggestion
// Pull-Request state at GitHub.
```

Reply:
github writes it as "pull request", I should follow the same spelling 
https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests